RELEASING.md

```diff
@@ -7,6 +7,7 @@
    and thus CI doesn't need to run,
    you can then add "[ci skip]" to the commit message.
 1. Tag the release: `git tag vVERSION`
+1. Sign the release using your GPG key
```

Comment:
The style here seems to be: punctuation at the end of each bullet point unless it ends with a code snippet.